emulators/qemu/Portfile

```diff
@@ -142,61 +111,61 @@ foreach t {i386 x86_64 alpha {arm aarch64} cris hppa lm32 m68k {microblaze micro
            nios2 moxie or1k {ppc ppc64} riscv32 riscv64 rx s390x {sh4 sh4eb} {sparc sparc64} tricore unicore32 {xtensa xtensaeb}} {
     variant target_[lindex $t 0] description "Add target support for [join $t {, }]" "append target_list \",[join $t -softmmu,]-softmmu\""
 }
-default_variants-append +target_i386 +target_x86_64
+default_variants-append +target_i386 +target_x86_64 +target_arm
```

Comment:
@kakuhen why intel macs shouldn't target ARM by default?

Reply:
For the same reason target_ppc isn't enabled by default. People who want the feature will enable it themselves, and this would conform to the "principle of least surprise."

Reply:
@kakuhen ppc target is obsolete and I can't say the same for intel target ;)

Reply:
@kakuhen Given that anything not listed in `default_variants` has to be built client-side from source, I think it's beneficial to include ARM by default, especially since it's now one of the contemporary CPU architectures supported by macOS.

For comparison, both FreeBSD's qemu port and homebrew's qemu formula include *all* targets by default, which we're not doing here (... but I think we ought to consider enabling them all in a future change).

Reply:
@catap OK looks like I was mistaken. I see that qemu-system-ppc is already provided by default, so targetting ARM makes more sense than not targetting it, from my (biased) perspective, at least. Because now a default install of QEMU will let you emulate every OS X version ;) For this reason, I approve the changes.

@landonf Good catch regarding FreeBSD ports! I used QEMU on FreeBSD briefly, and I didn't notice this at all.